boards/pba-d-01-kw2x/include/periph_conf.h

```diff
@@ -112,7 +92,21 @@ static const uart_conf_t uart_config[] = {
         .pcr_tx = PORT_PCR_MUX(3),
         .irqn   = UART2_RX_TX_IRQn,
     },
+    {
+        .dev    = UART0,
+        .clken  = (volatile uint32_t*)(BITBAND_REGADDR(SIM->SCGC4, SIM_SCGC4_UART0_SHIFT)),
+        .freq   = CLOCK_CORECLOCK,
+        .pin_rx = GPIO_PIN(PORT_D, 6),
```

Comment:
Maybe I'm missing the point, but isn't PORT D6 used by RBG (red) led? See also: https://github.com/PHYTEC-Messtechnik-GmbH/phynode-riot-examples/wiki/Interfaces